### DISCUSSION TITLE
tools git error

### DISCUSSION THREAD

**Discussion by RandyHoffpauir** [thoughtspot/cs_tools#91] [Q&A]
When I run this command for cs_tools:

cs_tools tools git branches commit --metadata-ids 69089aaa-cbbf-4f78-bca9-1039b3c15487 --branch-name development --comment "my updating comment" --delete-aware

I get the following error:

Error creating the configuration: <Response [400 Bad Request]>.
b'{"error":{"message":{"debug":{"code":15012,"incident_id_guid":"a3a7295f-7e10-4751-8760-8640a5e25537","debug":"Reposito
ry not in the correct state for the operation.Exception:Cannot pull into a repository with state: REBASING_MERGE"}}}}'.

It does not appear to be an issue with GitHub. I can commit and merge just fine. It seems like a thoughtspot issue but I don't know how to clear it.

**Comment by boonhapus** [comment]
Hey @RandyHoffpauir - I've got engineering looking into this one for you. We'll follow up on the Support ticket you opened. If this turns out to be something we can handle in CS Tools, I'll let re-open and tag you on this discussion.
